### ISSUE TITLE
Python 3 transition

### ISSUE THREAD

**Issue by krmaxwell** [krmaxwell/maltrieve#88]
`(venv)kmaxwell@newton:~/src/maltrieve(master)$ caniusepython3 -r requirements.txt 
Finding and checking dependencies ...
[WARNING] Stale overrides: set([u'python-neutronclient'])

You need 2 projects to transition to Python 3.
Of those 2 projects, 1 has no direct dependencies blocking its transition:

  gevent (which is blocking grequests)
`

**Comment by sroberts** [comment]
Are you really gonna do that? Really?!

**Comment by krmaxwell** [comment]
_...maybe..._

**Comment by wessorh** [comment]
lets all hope, not.

-rick

**Comment by krmaxwell** [comment]
Yeah, code that works across multiple versions is truly terrible.

**Comment by krmaxwell** [comment]
https://docs.python.org/2/howto/pyporting.html
